src/routes/signup.ts

```diff
@@ -38,14 +43,11 @@ router.post<{}, DataResponse, SignupRequest>('/signup', async (req, res, next) =
         return
     }
   
-    //FIXME: it seems that the jwt library has since been updated and this code no longer works
-    //const token = jwt.sign({ id: user.id }, process.env.JWT_SECRET as string, {
-    //  expiresIn: process.env.USER_TOKEN_EXPIRATION_TIME as string,
-    //});
-
-    //TODO: add token to the response data
-    //TODO: don't return the hashed password. That's just being lazy
-    response.data = { ...user };
+    const token = jwt.sign({ id: user.id, expiresIn: process.env.USER_TOKEN_EXPIRATION_TIME as string }, 
+                            process.env.JWT_SECRET as string);
+    //TODO: Add token to the whitelist
+    
+    response.data = {uuid: user.id, jwt: token}
```

Comment:
I think it's better just to say this is the `id` (identifier) or `uid` (user identifier)
```suggestion
    response.data = {id: user.id, jwt: token}
```